app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -1,5 +1,6 @@
 package com.github.h3lp3rs.h3lp
 
+import FireForum
```

Comment:
This import is weird. Are we sure the corresponding file is in the right package?

Reply:
Good point ! The file FileForum didn't have a package declaration